metrics/README.rst

```diff
@@ -7,6 +7,6 @@ This directory contains *metric definition* YAML files for each EUPS package tha
 Files are named after the EUPS package.
 For example: ``validate_drp.yaml`` for the `validate_drp`_ package.
 
-The format of these YAML files is *currently* documented in https://sqr-017.lsst.io/v/DM-9533/index.html#metric-yaml.
+The format of these YAML files is *currently* documented in https://sqr-017.lsst.io/index.html#metric-yaml.
```

Comment:
Are these files to end up in pipelines_lsst_io, and if so, how are they discovered?

Reply:
@SimonKrughoff, good point. Schema references and user guides for creating new metric and specification YAML definitions will end up in the `lsst.verify` user guide in pipelines.lsst.io. It'll go there since the format is defined in `lsst.verify` and `verify_metrics` just happens to be the default package for declaring metrics and specifications.

We'll deprecate both SQR-017 and SQR-019, which have mostly helped us communicate early development.